Makefile

```diff
@@ -14,6 +14,10 @@ setup:
 	uv venv
 	uv pip install -U -e .[dev]
 	. .venv/bin/activate
+	pipx install harlequin
```

Comment:
We could add `harlequin` to the project dev requiremets and avoid having to run this while keeping things contained.

https://github.com/davidgasquez/gitcoin-grants-data-portal/blob/main/pyproject.toml#L40